stubs/AMQPQueue.php

```diff
@@ -350,4 +362,14 @@ public function getChannel()
     public function getConnection()
     {
     }
+
+    /**
+     * Get latest consumer tag. If no consumer available or the latest on was canceled empty string will be returned.
```

Comment:
Yeah, but it adds more lines and few extra checks for this, and `is_empty()` and in `if` cases empty string will acts the same as null. And for now in `var_dump()` output we use empty string if no consumer tag set. But if you think `null` is more reasonable return value I will make such trivial changes.